projects/components/src/modal/modal.ts

```diff
@@ -36,15 +36,18 @@ export const getModalDimensions = (modalSize: ModalSize): ModalDimension => {
     case ModalSize.Large:
       return getModalDimensionObject(640, 720);
     case ModalSize.LargeTall:
-      return getModalDimensionObject(640, 800);
+      return getModalDimensionObject(640, '90vh');
```

Comment:
please ensure you test the change across various modals in the system today.

Reply:
I did check a few modals, but it shouldn't be impactful anyway. Just making a better use of space when it is available.